MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/labs/DashedLineDirectionsPickerActivity.java

```diff
@@ -117,11 +118,10 @@ public void onCameraIdle() {
    * to the place picker location
    */
   private void initDottedLineSourceAndLayer(@NonNull Style loadedMapStyle) {
-    dashedLineDirectionsFeatureCollection = FeatureCollection.fromFeatures(new Feature[] {});
-    loadedMapStyle.addSource(new GeoJsonSource("SOURCE_ID", dashedLineDirectionsFeatureCollection));
+    loadedMapStyle.addSource(new GeoJsonSource(SOURCE_ID));
     loadedMapStyle.addLayerBelow(
       new LineLayer(
-        "DIRECTIONS_LAYER_ID", "SOURCE_ID").withProperties(
+        "DIRECTIONS_LAYER_ID", SOURCE_ID).withProperties(
```

Comment:
nit "DIRECTIONS_LAYER_ID" could also be a static final String

Reply:
Fixed